libdnf/hy-repo.c

```diff
@@ -166,6 +172,12 @@ hy_repo_set_priority(HyRepo repo, int value)
 }
 
 void
+hy_repo_set_includes_enabled(HyRepo repo, gboolean use_includes)
+{
+    repo->includes_enabled = use_includes;
```

Comment:
I started with "use_includes". But renamed it later. And inconsistency arise :(
So, you prefer rename back to "use_includes" or sync to "includes_enabled" on all places?